app/javascript/react/screens/App/Overview/screens/MappingWizard/components/SourceClusterSelect/SourceClusterSelect.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import {
+  Form,
+  FormGroup,
+  Grid,
+  InputGroup,
+  FormControl,
+  ControlLabel,
+  DropdownButton,
+  MenuItem
+} from 'patternfly-react';
+
+const SourceClusterSelect = ({
+  form,
+  sourceClusters,
+  selectSourceCluster,
+  selectedCluster
+}) => {
+  const mappingObject = form.includes('Networks') ? 'networks' : 'datastores';
+
+  return (
+    <div className="source-cluster-select">
+      <Form horizontal>
+        <FormGroup>
+          <Grid.Col componentClass={ControlLabel} sm={6}>
+            {__(
+              sprintf(
+                'Map source %s to target %s for cluster',
```

Comment:
For proper i18n string extraction, this needs to be -
`sprintf(__('Map source %s to target %s for cluster', ...)`

i.e. the `__('some text')` pattern needs to exist